### ISSUE TITLE
Interference with other addons

### ISSUE THREAD

**Issue by CaioCMartinelli** [gdh1995/vimium-c-helpers#5]
Hello, I'm having compatibility problems with addons that use unique keys to execute commands in Firefox.

The biggest problem (the one that affects me most and i really care) is the "Video Speed Controller [1]", as the name says, a manager with media controls for any HTML5 video on web.

When using Vimium, it overlaps and completely prevents the HTML5 configured media buttons from working, is there anything that can be done to avoid this interference?

[1] https://addons.mozilla.org/pt-BR/firefox/addon/videospeed/
[1] https://github.com/codebicycle/videospeed

**Comment by gdh1995** [comment]
do you mean Vimium C, but not Vimium-fx affect it (they're different extensions)?


could you give a snapahot of such a case?



---Original---

**Comment by gdh1995** [comment]
Um, seems you were not saying about "a button covers another", but wants to make Vimium C ignore `z/s/d/x/v` and let other extensions control videos. Am I right this time?

Then you may open Vimium C Options page and:
* go to `custom key mappings` and unmap such keys
* or go to `exclusion rules` and specify some patterns and keys, and then Vimium C will ignore them on matched websites.

**Comment by CaioCMartinelli** [comment]
> Um, seems you were not saying about "a button covers another", but wants to make Vimium C ignore `z/s/d/x/v` and let other extensions control videos. Am I right this time?
> 
> Then you may open Vimium C Options page and:
> 
>     * go to `custom key mappings` and unmap such keys
> 
>     * or go to `exclusion rules` and specify some patterns and keys, and then Vimium C will ignore them on matched websites.

Yes, i tried disable Vimium-C on Youtube and unmap all keys but neither worked. If i enable Vimium-C (I tried Vimium-FF too) on extensions, the addon that controls HTML5 media imediately stops responding for any configured key, as if it had stopped receiving the information about the key being pressed.

Is there something more that i can do?

![image](https://user-images.githubusercontent.com/72528065/95416940-3648dd80-090a-11eb-9cd4-5152b5455ebb.png)

![image](https://user-images.githubusercontent.com/72528065/95417010-642e2200-090a-11eb-9cf9-72b8874678bb.png)
